### ISSUE TITLE
In-game Overlay only working in Windowed/borderless mode

### ISSUE THREAD

**Issue by williams57** [Piotrekol/StreamCompanion#395]
As says above...idk if that's just how it is and there's no way to make it run on Full screen, but wanted to check just in case

**Comment by Piotrekol** [comment]
That's not "how it is" - it working in fullscreen is the main point.
Are you sure that it is simply not displaying these offscreen when fullscreened?

This is first time I've heard it works in windowed mode and not fullscreen tbh..

**Comment by williams57** [comment]
unless something is very messed up with my settings, the tracker doesnt show up for me in full screen...the resolution is the same for both, as well as the positioning, but it just doesnt pop up...

I swap to windowed/borderless and it pops up every time so no idea what to do there...i dont want to have to run it in windowed since that greatly messes with my pc performance

**Comment by williams57** [comment]
I also grabbed the Trace logs to see whats going on with it but nothing is making sense...

[2023-10-05.txt](https://github.com/Piotrekol/StreamCompanion/files/12818214/2023-10-05.txt)

**Comment by Piotrekol** [comment]
dual gpus depending on config could definitely be an issue - I tested intel igpu/radeon mix the past, but never nvidia.
Is there some other overlay running, that is only activating while in fullscreen? Note that overlays may remain activated even when not displaying anything - you may have to uninstall that other application to test.

**Comment by williams57** [comment]
I don't believe so, I've disabled all the overlays I can think of (Discord, Overwolf, etc.) but if its something built in that I've never used before then I have no idea...
